bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.1.1
+version: 8.1.2
```

Comment:
It is adding a non-breaking feature so I believe it should be a minor version bump.

Reply:
OK